src/layouts/LoginLayout.vue

```diff
@@ -32,7 +32,12 @@ export default {
   },
   methods: {
     getImageUrl() {
-      return location.origin + location.pathname + 'img/login-company-logo.svg';
+      let pathName = location.pathname !== '/' ? location.pathname : '';
```

Comment:
This condition is placed  because in case of ASM being launched via HMC location.pathname was required and other places it would just come as '/' which would not let the image to load. Hence the ternary operator.